src/test/kotlin/no/nav/helse/flex/periode/PeriodeTest.kt

```diff
@@ -0,0 +1,61 @@
+package no.nav.helse.flex.periode
+
+import no.nav.helse.flex.domain.Periode
+import org.amshove.kluent.`should be equal to`
+import org.junit.jupiter.api.Test
+import java.time.LocalDate.of
+
+class PeriodeTest {
+    @Test
+    fun testerUkedager() {
+        val periode =
+            Periode(
+                of(2024, 10, 21),
+                of(2024, 10, 27),
+            ) // mandag til søndag
```

Comment:
Jeg tenker kanskje at dette kan være forskjellige tester i stedet for å ha det som kan være navnet på testen som en kommentar?